source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,118 @@
+---
+title: RSpec 3.6 has been released!
+author: Sam Phippen, TODO: other editors
+---
+
+RSpec 3.6 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be an easy
+upgrade for anyone already using RSpec 3, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes over XXX commits and YYY
+merged pull requests from over 50 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+### Core: Errors outside examples now handled and formatted well
+
+In previous versions of RSpec, we allowed errors encountered while loading spec
+files or running `:suite` hooks to crash the ruby interpreter, giving you its
+default full-stacktrace output.  In RSpec 3.6, we now handle all errors that
+occur outside examples, and format them nicely including a filtered backtrace
+and a code snippet for the site of the error.  For example, an error in a
+`before(:suite)` hook is now formatted like this:
+
+<img width="479" alt="screen shot 2017-04-27 at 9 59 12 pm"
+src="https://cloud.githubusercontent.com/assets/49391/25514870/cb9db6c8-2b94-11e7-952f-f26fd783512b.png">
+
+Thanks to Jon Rowe for assisting with this feature.
+
+### Core: `config.fail_if_no_examples`
+
+As it currently stands RSpec will exit with code `0` indicating success if no
+examples are defined. This configuration option makes it possible to cause RSpec
+to exit with code `1` indicating failure. This is useful in CI environments, it
+helps detect when you've misconfigured RSpec to look for specs in the wrong
+place or with the wrong pattern. When this occurs we consider finding no specs
+to being an error, as opposed to always passing.
+
+~~~ ruby
+RSpec.configure do |config|
+  config.fail_if_no_examples = true
+end
+~~~
+
+A special thanks to Ewa Czechowska for getting this in to core.
+
+### Core: Output coloring is set automatically if RSpec is outputting to a tty.
+
+In past versions of RSpec, you were required to specify `--color` if you wanted
+output coloring, regardless of whether you were outputting to a terminal, a
+file, a CI system, or some other output location. Now, RSpec will automatically
+color output if it detects it is running in a TTY. You can still force coloring
+with `--color`, or if you are running in a TTY and explicitly do not want color,
+you can specify `--no-color` to disable this behavior.
+
+For existing RSpec projects, that were initialized with either `bundle exec
+rspec --init` or `bundle exec rails g rspec:install` you can find this setting
+in your `.rspec` file if you'd like to change it.
+
+We thank Josh Justice for adding this behavior to RSpec.
+
+### Expectations: Scoped aliased and negative matchers
+
+TODO: is anyone more familiar with this feature who could full this out?
```

Comment:
Basically, the idea is that you can now call `alias_matcher` or `define_negated_matcher` in an example group to limit the scope of the newly defined matchers to just that group, rather than them being defined globally for use from all groups.